### DISCUSSION TITLE
Energy Storage Default Cost of Storage

### DISCUSSION THREAD

**Discussion by adam-morse** [NREL/REopt-API-Analysis#152] [5 - Financial Input Questions]
I'm getting tripped up on the default power capacity costs ReOpt is using. $910/kW is much higher than we've ever used. I can't view the Lazards report mentioned in the manual, but the powerpoint version of the report indicates more typical costs of $50-100/kWAC. Am I missing something expecting the energy capacity cost to be much higher than power capacity?

**Comment by adfarth** [comment]
Hello, the default value calculation is shown below (screenshot from the R[Eopt User Manual ](https://reopt.nrel.gov/tool/reopt-user-manual.pdf#page=5). Note that the default assumes a 2-hour battery. 

![image](https://github.com/user-attachments/assets/f0aaddef-fec4-4052-bd79-12c2361ef4af)
